.github/workflows/tox.yml

```
@@ -37,27 +37,12 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                python: [3.9]
-                django: [31, 32, main]
+                python: ["3.10"]
+                django: [32, 40, 41, main]
```

I'd do lowest/highest pythons, then latest for each supported major version of Django, plus main.

```suggestion
                python: ["3.8", "3.11"]
                django: [32, 41, main]
```